src/index.js

```diff
@@ -0,0 +1,149 @@
+import './style.css';
+
+const todoArr = [];
+/* eslint-disable no-use-before-define */
+const addTodoItem = () => {
```

Comment:
- [ ] You want to consider implementing some to-do tasks in the array as the requirements said. ⚒️ 

![to-do-objects](https://user-images.githubusercontent.com/79876959/145333878-b84ad70b-6a2c-4c92-bbd3-019424b75057.PNG)